src/amo/components/VPNPromoBanner/index.js

```diff
@@ -203,8 +215,15 @@ export class VPNPromoBannerBase extends React.Component<InternalProps, State> {
   }
 }
 
-const mapStateToProps = (state: AppState): PropsFromState => {
+const mapStateToProps = (
+  state: AppState,
+  ownProps: InternalProps,
+): PropsFromState => {
+  const { slug } = ownProps.match.params;
+  const addon = getAddonByIdInURL(state.addons, slug);
+
   return {
+    addonIdentifier: (addon && String(addon.id)) || slug,
```

Comment:
I don't understand why we'd pass the `addon.id` instead of the `slug` when the add-on is loaded? IIUC we would pass the actual slug when the add-on isn't yet loaded.

Reply:
I think our general approach is ti not use the slug for reporting purposes as it can change. We use the `addonId` in other instances where we are passing an add-on identifier to GA. In the case of this promo, we should theoretically always have an add-on loaded, as this component is only rendered if we have an add-on loaded. Using the slug in the case where an add-on is not loaded is just a safety net in case that somehow happens.

Reply:
@willdurand As per our discussion on matrix, I have changed the code so that we only pass `utm_content` if an add-on is loaded, and in that case we use the `addonId`.